### DISCUSSION TITLE
Is it possible to obtain all queues?

### DISCUSSION THREAD

**Discussion by nullndr** [taskforcesh/bullmq#1226] [Q&A] [answered]
I know that given a queue is it possible to obtain his jobs with the `getJobs` methods, but is there a way to get all queues on Redis?

I tried to parse all keys in Redis to get all queues names and then create a new instance of each one but it looks like dirty code to me.

**Comment by manast** [comment, marked as answer]
There is currently not a queue registry in BullMQ, but if you use a regexp searching for the :id key (the one holding the id counter for every queue) you can get a list pretty quickly.

**Reply by nullndr** [reply]
Thank you so much
